manifests/server/role.pp

```diff
@@ -206,6 +206,7 @@
           environment => $environment,
         }
       } elsif ($dialect == 'redshift') {
+        warning('UNLESS clause support is not yet supported for setting password on Redshift users')
```

Comment:
Same comment. Can you include a comment with a pointer to what specifically blocks this support?

Reply:
I describe directly below this line that this is caused by the lack of pg_shadow access. I can move the comment up or into the warning statement if that works better.

Reply:
I describe directly below this line that this is caused by the lack of pg_shadow access. I can move the comment up or into the warning statement if that works better.